### PR TITLE
Cove renaming

### PR DESCRIPTION
**Pull request by atishp04**
1st set of changes that renames the references of TEE* to COVE in the sbi spec & final output spec.
Future PRs will update the other references.

### CODE REVIEW

**Comment by atishp04** [comment]
Most of these changes are mechanical and have minimal rewording of the existing text. It would be great if we can merge this one first. I will send out the RFC once this is done so that upstream community don't get confused with mixed naming.

I am trying to modify other "AP-TEE" name usage in the spec to best of my ability. That may require more discussion. 
I will push a second PR for that which can be merged later.

File: specification/sbi_cove.adoc [hunk 1]
```diff
@@ -1816,113 +1816,113 @@ This interface specification is TBD.
...
+| <<sbi_covi_rebind_aia_imsic_end, sbi_covi_rebind_aia_imsic_end>> |
 Completes the rebind process for the specified vCPU from this physical CPU and its guest
 interrupt files. Must be called from the same physical CPU as
-`sbi_tee_interrupt_rebind_aia_imsic_begin()`.
+`sbi_covi_rebind_aia_imsic_begin()`.
 
 |===
 
-=== Summary of TEEI - TG-ABI
+=== Summary of COVE Guest Extension (COVG)
```
Comment (rsahita) [inline specification/sbi_cove.adoc:1864]:
```suggestion
=== Summary of CoVE Guest Extension (COVG) 
```

File: specification/sbi_cove.adoc [hunk 2]
```diff
@@ -1816,113 +1816,113 @@ This interface specification is TBD.
 
 |===
 
-=== Summary of TEE Interrupt Extension
+=== Summary of COVE Interrupt Extension(COVI)
```
Comment (rsahita) [inline specification/sbi_cove.adoc:1819]:
```suggestion
=== Summary of CoVE Interrupt Extension(COVI)
```

File: specification/sbi_cove.adoc [hunk 3]
```diff
@@ -1672,73 +1672,73 @@ Both `cert_request_addr`, `request_data_addr` and `cert_addr_out` must be 4kB-al
 
 
 
-== Summary Listing of TEEI
+== Summary Listing of CoVE functions 
 
-=== Summary of TEEI - TH-ABI
+=== Summary of COVE Host Extension (COVH)
```
Comment (rsahita) [inline specification/sbi_cove.adoc:1677]:
```suggestion
=== Summary of CoVE Host Extension (COVH)
```

File: specification/sbi_cove.adoc (hunk 1 above)
Comment (atishp04) [reply specification/sbi_cove.adoc:1864]: Do you prefer to remove all references to T[GH]-ABI ? There are bunch of them including few in the figures. 
Same goes for TEEI - TEE Interface - Should we get rid of that acronym as well ?

Comment (atishp04) [reply specification/sbi_cove.adoc]: Done

File: specification/sbi_cove.adoc [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+== Confidential VM Extension (CoVE) SBI extension proposal
+This section describes the normative Confidential VM Extension(CoVE) SBI extension
+proposal.  The proposal introduces three new extensions that will be described later:
 
-* TEE Host Extension (EXT_TEE_HOST), 
-* TEE Interrupt Extension (EXT_TEE_INTERRUPT), and 
-* TEE Guest Extension (EXT_TEE_GUEST).
+* CoVE Host Extension (EXT_COVE_HOST), 
+* CoVE Interrupt Extension (EXT_COVE_INTERRUPT), and 
+* CoVE Guest Extension (EXT_COVE_GUEST).
```
Comment (rsahita) [inline specification/sbi_cove.adoc:10]: should these be EXT_COVH, EXT_COVI and EXT_COVG?

**Review by rsahita — CHANGES_REQUESTED** [review]

File: specification/sbi_cove.adoc (hunk 4 above)
Comment (atishp04) [reply specification/sbi_cove.adoc:10]: Fixed.

**Review by rsahita — APPROVED** [review]

**Pull request merged**